java/Nikiforov-Oleksandr/src/com/zoolatech/lecture1/tasks/_4/Task4.java

```diff
@@ -0,0 +1,41 @@
+package com.zoolatech.lecture1.tasks._4;
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts a day of the week as an integer number (1 - Monday, 2 - Tuesday, …, 7 - Sunday)
+ * and output “Need to go to work…” if the day is a weekday or “Sleeping…” if it’s a weekend.
+ * Create two versions of the program: one should use the if-else statement, second - the conditional operator.
+ */
+
+public class Task4 {
+    public static void main(String[] args) {
+        // task solution code here
+        Scanner sc = new Scanner(System.in);
+        int numberofDay = sc.nextInt();
```

Comment:
`numberOfDays`

Reply:
I changed the name of the variable, I don't quite understand what's wrong

Reply:
`Of` starts from the uppercase letter

Reply:
Changed